### PR TITLE
So close...but so far

### PR DESCRIPTION
**Pull request by mfmeier**

### CODE REVIEW

**Comment by mfmeier** [comment]
it gives the answer I want, but it doesnt give a boolean response so it fails

File: lib/most_frequent_integer.rb [hunk 1]
```diff
@@ -3,5 +3,8 @@
 # exists, return nil.
 
 def find_most_frequent_integer(arr)
-  # put your code here
+    num_val = Hash.new(0)                   #This is the new empty hash
+    arr.each {|arr| num_val[arr] += 1}      #Here we iterate over the array, puting it into a hash with the key being the number and the value how many
+    num_val.key(num_val.values.max)         #Here we iterate over the hash and find the key with the highest value
 end
+find_most_frequent_integer(arr)
```
Comment (sumeetjain) [inline lib/most_frequent_integer.rb]: You should remove this line. It's causing an error that's preventing `rake` from running, so you aren't able to run the tests.

I think your code still needs a small change after that, but it's definitely close.

**Comment by mfmeier** [comment]
I am STILL having trouble with Cloud 9 (not that it really matters with this being my last problem to finish up and us not using Cloud 9 soon).  I need to tell this program to 1) return nill if there is more than one max.value (but it just displays the first one) and 2) return nill if all values are one.  Should I try to build an if then statement that would 'return nil' or can you help me with this?

**Comment by sumeetjain** [comment]
For Cloud9, try running `bundle install` and then seeing if other commands start working.

---

> Should I try to build an if then statement that would 'return nil' or can you help me with this?

Yeah, that's the right idea. I can help you with it, but your idea is correct. Want to try building it and then seeing if you have specific questions?

**Comment by sumeetjain** [comment]
:+1:

**Pull request closed without merging**